### ISSUE TITLE
Trouble Migrating to New Server/URL

### ISSUE THREAD

**Issue by gkobilansky** [valendesigns/option-tree#277]
Using the Hope Responsive Theme (http://themeforest.net/item/hope-responsive-wordpress-charity-theme/5747208) which includes OptionTree, for a client. 

I'm running into trouble when migrating from localhost to an online environment. Here are the steps to reproduce:
1. Migrate WP files and database
2. Use the search and replace plugin (https://interconnectit.com/products/search-and-replace-for-wordpress-databases/) to change URLs in the database
3. Use OptionTree Export and Import to copy Settings, Theme Options and Layouts

-- At this point I'm getting close to the site working, but OptionTree assets point to localhost (logo, sliders and other images) and font styles aren't correct on most pages. --
1. If I use the Search and Replace plugin again to change the OptionTree data to point to production URL in wp_options, I lose all the settings. 

There's got to be something I'm missing, but my google-fu hasn't turned up anything around migrating OptionTree between different URLs.

**Comment by djanaweb** [comment]
Hi there I have a similar problem,

I bought a theme which has the option tree included (an older version 2.3.2) with the current Wordpress 4.1.
When the developer migrated the site to it's new domain, everything worked except for Option tree : the configuration had been reset and all the changes made erased. Also only the "General Settings" shows correctly, all the other tabs (fonts etc) when clicked display a blank page.
I've seen many people have had similar issues, we've tried using the interconnect to change the urls, but it seems not to change anything.
We'll give the Fix sterilization a go but as we used the interconnect it shouldn't change much should it?

As I'm no developer I also tried fixing it by following some of these advices : 
Deactivating all other plugins (cache etc), as seen on another thread, with no success.
Another thread suggested to "Go to the User Profile view for the account you use for editing. Below the "Password" section is the OptionTree setting, enable the "Show Settings and Docs". But I only have "Theme options in my back office, because it was included in the theme.

I also have a silly question: how do I update the plugin? Should I replace the option-tree folder with the newer version? Because I can't change it through the back office. Wouldn't that erase the current settings?

Thank you for your help,
